.github/workflows/pr.yml

```diff
@@ -22,6 +22,4 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-gradle-
     - name: Build
-      env:
-        GRADLE_OPTS: -Xmx6g -Xms6g -Dorg.gradle.daemon=false
```

Comment:
I don't think this heap is necessary, gradle emits a warning message if it runs out of cache space and that is not showing up in any builds aside from the release build

Reply:
actually there is a magical blend of `-Xmx and org.gradle.daemon` here that makes the build not fork off a daemon

6g is definitely way more than we need but this setting is effective/useful so I'll put it back in with a smaller heap

Reply:
I've moved this to a top level env block in all the actions